packages/repository-tests/src/crud/relations/acceptance/multi-relations-inclusion-resolver.relation.acceptance.ts

```diff
@@ -63,15 +70,22 @@ export function hasManyInclusionResolverAcceptance(
       const parent = await customerRepo.create({name: 'parent'});
       const customer = await customerRepo.create({
         name: 'customer',
-        parentId: parent.id,
+        //parentId: parent.id,
+      });
+      const address = await addressRepo.create({
+        street: '8200 Warden',
+        city: 'Markham',
+        province: 'On',
+        zipcode: '8200',
+        customerId: parent.id,
       });
       const order = await orderRepo.create({
         description: 'an order',
         customerId: parent.id,
       });
 
       const result = await customerRepo.find({
-        include: [{relation: 'orders'}, {relation: 'customers'}],
```

Comment:
While I was fixing the CI failure, I think the flag might somehow break  how we deal with `ObjectId` in MongoDB. Before we were able to traverse related instances with `ObjectId`.  Not sure it has something to do with the type.